LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -2958,27 +3099,18 @@
     this.portalAPI.setCSRFToken(CSRFToken);
 
 
-
     // Fix dialog boxes not going away when clicking outside of box
-    $(document).mouseup(function(e) {
+    $(document).on('mousedown', function(e) {
```

Comment:
This caught my attention as this is for 508 compliance.  Someone who is blind isn't going to be using a mouse.  So... do we need to come up with a composite stimulus check?

Reply:
These dialogs are also set up to close on escape, and also with a new more obvious close button, so this should still be ok.